### ISSUE TITLE
[Accessibility] Horizontal diagrams don't convey meaningful information to screen reader users

### ISSUE THREAD

**Issue by PeterSaathoff-Harshfield** [ampproject/amp.dev#4502]
Environment:
Any OS/browser and screen reader (such as VoiceOver, NVDA, or ChromeVox)

Steps to reproduce:
1. Open https://amp-dev-staging.appspot.com/page-experience/
2. Turn on a screen reader
3. Linear navigate with the screen reader to the URL field, enter a URL, and press Analyze
4. Navigate with the screen reader to the horizontal diagrams under the "Core Web Vitals" heading, in both the Field data and Lab data tabs

An example of how a screen reader reads a Field data diagram:
Page loads
66% passed
75th percentile (complimentary)

How it reads a Lab data diagram:
Page loads
6 s
4 s
2.5s
11.4 s (complementary)

These numbers, without spatial context, are not meaningful to screen reader users. They are navigated out of order with their visual order. It's also unclear why the number at the right end of each diagram is marked as a complimentary landmark role.

The information conveyed by these diagrams should be conveyed in a way that also works for screen reader users who will consume the information linearly, and colorblind users who don't derive meaning from different colors.

[WCAG Success Criterion 1 Perceivable](https://www.w3.org/TR/WCAG21/#perceivable)
[WCAG Success Criterion 1.3.1 Info and Relationships](https://www.w3.org/TR/WCAG21/#info-and-relationships)
[WCAG Success Criterion 1.3.2 Meaningful Sequence](https://www.w3.org/TR/WCAG21/#meaningful-sequence)
[WCAG Success Criterion 1.3.3 Sensory Characteristics](https://www.w3.org/TR/WCAG21/#sensory-characteristics)

![image](https://user-images.githubusercontent.com/30604285/92971853-10790780-f436-11ea-94a6-a70a24b9f51a.png)

![image](https://user-images.githubusercontent.com/30604285/92971864-166ee880-f436-11ea-9a70-01d89f2ebb5a.png)

**Comment by caroqliu** [comment]
@nainar What do you think about for SRs to hide the visual purely as decorative with the following additional points of reference:
- `Your score: ` is updated to `Your score at the 75th percentile: `
- `Opportunity to improve: ` for field data is updated to `Passing pages: X%`
- Make the Good and Needs Improvement thresholds available in the info text, i.e. "Page scores of <= 2.5s are Good, 2.5–4s are Needs Improvement, and >4s are Poor."

@PeterSaathoff-Harshfield Is it OK if the diagram is hidden from screen readers and inaccessible to colorblind users as long as the same info is available through the updates mentioned above?

**Comment by PeterSaathoff-Harshfield** [comment]
This looks like a good solution.
I see blue text in your screenshot, just want to make sure that meets the 4.5:1 (or 3:1 if it's large enough) contrast ratio.

**Comment by caroqliu** [comment]
Thanks for the feedback! Yep, the blue is 5.64:1 contrast ratio.

I'll make these changes now.
